autometa/common/external/prodigal.py

```diff
@@ -255,57 +281,63 @@ def orf_records_from_contigs(contigs, fpath):
         Why the exception is raised.
 
     """
-    version = get_versions('prodigal')
-    if version.count('.') >= 2:
-        version = float('.'.join(version.split('.')[:2]))
+    version = get_versions("prodigal")
+    if version.count(".") >= 2:
+        version = float(".".join(version.split(".")[:2]))
     else:
         version = float(version)
 
     records = []
-    for record in SeqIO.parse(fpath, 'fasta'):
+    for record in SeqIO.parse(fpath, "fasta"):
         if version < 2.6:
-            orf_id = record.description.split(
-                '#')[-1].split(';')[0].strip().replace('ID=', '')
-            contig_id = record.id.replace(f'_{orf_id}', '')
+            orf_id = (
+                record.description.split("#")[-1]
+                .split(";")[0]
+                .strip()
+                .replace("ID=", "")
+            )
+            contig_id = record.id.replace(f"_{orf_id}", "")
         else:
-            contig_id = record.id.rsplit('_', 1)[0]
+            contig_id = record.id.rsplit("_", 1)[0]
         if contig_id not in contigs:
             continue
         records.append(record)
     return records
 
 
 def main(args):
-    if args.verbose:
-        logger.setLevel(logger.DEBUG)
     nucls_out, prots_out = run(
         assembly=args.assembly,
         nucls_out=args.nucls_out,
         prots_out=args.prots_out,
         force=args.force,
         cpus=args.cpus,
-        parallel=args.parallel)
-    logger.info(
-        f'written:\nnucls fpath: {nucls_out}\nprots fpath: {prots_out}')
+        parallel=args.parallel,
+    )
+    logger.info(f"written:\nnucls fpath: {nucls_out}\nprots fpath: {prots_out}")
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
This whole block should be moved under `def main` to comply with `template.py`

Reply:
Another nice find. What a silly oversight. 🤦